script.js

```diff
@@ -1,10 +1,13 @@
 const startButton = document.querySelector('#startGameButton')
 const splashScreen = document.querySelector('#displaySplashInstructionsContainer')
 const avatar = document.getElementById('avatar')
+const gameTimer = document.querySelector('#stopwatchSecondsDisplay')
```

Comment:
More semantic name plz - something to make it obvious that this is the display of the gameTimer, not the timer itself